css/portal-dashboard/response-details/popup-class-nav.less

```diff
@@ -138,5 +138,50 @@
         color: @cc-charcoal;
       }
     }
+
+    .refreshSortContainer {
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      margin: 10px 0 0 20px;
+      cursor: pointer;
```

Comment:
I don't think you want this cursor on the container, but rather on the button.  Otherwise you get the weird affect where the cursor changes to the finger on the text (which isn't clickable) but not the button itself.  So delete `cursor: pointer;` above....